From ef7ce41b8f5091d9a4d72d27611a38ba1b1ec3c4 Mon Sep 17 00:00:00 2001
From: wu xia <xia.wu@intel.com>
Date: Sun, 11 Feb 2018 17:06:29 +0800
Subject: [PATCH 0975/1240] v4l2-control: add V4L2 control ID for Iris control

[Issue]
Support Iris control

[Root Cause]
Add V4L2 control ID for Iris control

[Changes]

[Test Platform]
Leafhill

Signed-off-by: Ding Yunliang <yunliang.ding@intel.com>
Signed-off-by: wu xia <xia.wu@intel.com>
---
 include/uapi/linux/v4l2-controls.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/uapi/linux/v4l2-controls.h b/include/uapi/linux/v4l2-controls.h
index 0819484..a329881 100644
--- a/include/uapi/linux/v4l2-controls.h
+++ b/include/uapi/linux/v4l2-controls.h
@@ -895,6 +895,8 @@ enum v4l2_jpeg_chroma_subsampling {
 #define V4L2_CID_MIPI_LANES			(V4L2_CID_IMAGE_SOURCE_CLASS_BASE + 64)
 #define V4L2_CID_WDR_MODE			(V4L2_CID_IMAGE_SOURCE_CLASS_BASE + 65)
 #define V4L2_CID_PWM_DUTY			(V4L2_CID_IMAGE_SOURCE_CLASS_BASE + 66)
+#define V4L2_CID_IRIS_MODE			(V4L2_CID_IMAGE_SOURCE_CLASS_BASE + 67)
+#define V4L2_CID_IRIS_STEP			(V4L2_CID_IMAGE_SOURCE_CLASS_BASE + 68)
 
 /* Image processing controls */
 
-- 
2.7.4

